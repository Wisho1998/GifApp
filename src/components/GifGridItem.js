import React from 'react'

const GifGridItem = ({title,url}) => {
  return (
    <figure className='card'>
        <img className='card__image' src={url} alt={title}/>
        <figcaption className='card__caption'>{title}</figcaption>
    </figure>
  )
}

export default GifGridItem;