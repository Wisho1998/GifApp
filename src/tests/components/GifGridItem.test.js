import React from 'react';
import { shallow } from 'enzyme';

import GifGridItem from '../../components/GifGridItem';

describe("Tests in <GifGridItem />", () => {
    test('Debe de mostrar el componente correctamente', () => { 
        const wrapper = shallow(<GifGridItem />)
        expect( wrapper ).toMatchSnapshot();
     })
})