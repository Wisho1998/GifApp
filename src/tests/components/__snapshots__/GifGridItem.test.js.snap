// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests in <GifGridItem /> Debe de mostrar el componente correctamente 1`] = `
<figure
  className="card animate__animated animate__fadeIn"
>
  <img
    className="card__image"
  />
  <figcaption
    className="card__caption"
  />
</figure>
`;
